mobile/src/views/VerificationView.jsx

```diff
@@ -55,9 +55,7 @@ const VerifcationView = ({navigation}) => {
             'Content-Type': 'multipart/form-data',
           },
         })
-          .then(result => {
-            return result.json()
-          })
+          .then(result => result.json)
```

Comment:
Should it be `result.json()` with the brackets?

Reply:
Yea, sorry I took the last comment too literal. Should have brackets now.